SemiTruck.cpp

```diff
@@ -0,0 +1,15 @@
+#include "SemiTruck.h"
+
+SemiTruck::SemiTruck(const std::string& n) : Vehicle(n) {}
+
+void SemiTruck::setSpeed(int s)
+{
+    std::cout << "\nHighway is adjusting " << name << "'s speed to " << s << " mph" << std::endl;
+    speed = s;
+}
```

Comment:
This function doesn't do anything different from the base class's implementation. 
There is no reason to override it.
Your job was to write 2 new functions.
```
    5) Add a Truck type
        semi-trucks don't evade when they're pulled over, unlike cars and motorcycles.  
        They'll also need a member function that can be called when they are added to the highway.  
         Look at the Highway member functions for more hints on this.
```
- [ ] revise